tests/test_utils.py

```diff
@@ -126,3 +127,187 @@ def test_create_segmented_index(starts, ends):
             assert index.get_level_values(
                 audformat.define.IndexField.END
             ).tolist() == [pd.NaT] * len(starts)
+
+
+@pytest.mark.parametrize(
+    'signal, sampling_rate, win_dur, hop_dur, expected',
```

Comment:
You never specified a `sampling_rate` other than 1, so maybe remove it from the parameters?

Reply:
Yes we can do it, but let's do it as a last step once we are happy with the tests and don't see the need to test other sampling rates.

Reply:
We still have only `1` as value. But I guess it will make it easier to extend the test at a later stage if we stick with `sampling_rate` as parameter here.